server/server.js

```diff
@@ -237,7 +237,7 @@ app.post(`${apiBaseUrl}/new-hired-participant`,
       [participantInfo.preferredLocation] = user.regions;
       participantInfo.crcClear = 'yes';
       participantInfo.interested = 'yes';
-      participantInfo.callbackStatus = 'false';
+      participantInfo.callbackStatus = false;
```

Comment:
Great catch - I wouldn't have been able to see this problem in the UI due to the participant already being hired 😄